thirdparty/misc/rgb2yuv.h

```diff
@@ -0,0 +1,41 @@
+#ifndef RGB2YUV_H
+#define RGB2YUV_H
+
+#include "core/typedefs.h"
```

Comment:
The source of this file needs to be documented in `thirdparty/README.md` and its copyright in `COPYRIGHT.txt`.

If you wrote it yourself, it shouldn't be in `thirdparty`.

Reply:
I inherited it from the original PR. It looks like it comes [stackoverflow](https://stackoverflow.com/a/9467305). I'll document it.

Reply:
Please, see comment in the PR.